src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,44 @@
+package domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static util.Exceptions.DUPLICATED_NAME_EXCEPTION;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<String> names) {
+        validateDuplicatedNames(names);
+        cars = convertNamesToCars(names);
+    }
+
+    private void validateDuplicatedNames(List<String> names) {
+        if (names.size() != Set.copyOf(names).size()) {
+            throw new IllegalArgumentException(DUPLICATED_NAME_EXCEPTION.getMessage());
+        }
+    }
+
+    public List<Car> getCars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    private List<Car> convertNamesToCars(List<String> names) {
+        List<Car> cars = new ArrayList<>();
+
+        for (String name : names) {
+            cars.add(new Car(name));
+        }
+
+        return cars;
+    }
+
+    public int getMaxForward() {
+        return cars.stream()
+                .sorted(Car::compareTo)
+                .toList()
+                .get(0)
+                .getForward();
```

Comment:
```suggestion
        return cars.stream()
                .min(Car::compareTo)
                .orElseThrow()
                .getForward();
```
위처럼 사용할 수도 있습니다.
`get(0)`과 같이 인덱스로 접근하는 방식은 `IndexOutOfBoundsException`을 발생시킬 수 있습니다.

Reply:
물론 더 좋은 방법은 compareTo를 사용하지 않는 것입니다 🥲

Reply:
아무도 움직이지 않으면 0이라 생각해 max 값이 존재하지 않는 경우까지 고려하지 못했습니다 반영 완료했습니다 👍